lib/widgets/pages/home_page.dart

```diff
@@ -113,25 +77,3 @@ const _bottomNavBarItems = [
     label: Strings.settingsNavTitle,
   ),
 ];
-
-void addTestDBWidget(BuildContext context) {
```

Comment:
Also wrong placement. Contact me and lets implement the test DB widget the proper way.